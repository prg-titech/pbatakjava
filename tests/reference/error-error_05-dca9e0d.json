{
    "basename": "error-error_05-dca9e0d",
    "cmd": "{infile}",
    "infile": "tests/error_05/build.sh",
    "infile_hash": "b17e967885b01189e3532569714c6e7f3158b6cae4f5e3c6481556ac",
    "stdout": null,
    "stdout_hash": null,
    "stderr": "error-error_05-dca9e0d.stderr",
    "stderr_hash": "f3974328983eef01e910a021e83e6295db28ab6323f7c0b882b1ae2d",
    "returncode": 4
}