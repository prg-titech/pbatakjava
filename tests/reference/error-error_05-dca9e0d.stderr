Fatal exception:
java.lang.Error: no solution found in component [Test v.1]
	at org.extendj.ast.Program.solve(Program.java:856)
	at org.extendj.ast.Program.solve(Program.java:899)
	at org.extendj.ast.BatakjavaBackend.run(BatakjavaBackend.java:132)
	at org.batakjava.BatakjavaCompiler.run(BatakjavaCompiler.java:62)
	at org.batakjava.BatakjavaCompiler.main(BatakjavaCompiler.java:31)
